decidim-forms/app/assets/javascripts/decidim/forms/forms.js.es6

```diff
@@ -31,4 +31,31 @@
       wrapperField: $(el)
     })
   });
+
+  const $form = $("form.answer-questionnaire");
+  if ($form.length > 0) {
+    $form.find("input").on("change", () => {
```

Comment:
```suggestion
    $form.find("input").on("change", () => {
```

Shouldn't this include textareas and selects as well?

Reply:
Ah, that's right, I'll check it out, thanks!